tests/test_examples.py

```diff
@@ -40,6 +40,8 @@
     'hmm.py --num-steps=1 --truncate=10 --model=3',
     'hmm.py --num-steps=1 --truncate=10 --model=4',
     'hmm.py --num-steps=1 --truncate=10 --model=5',
+    'hmm.py --num-steps=1 --truncate=10 --model=6',
```

Comment:
Could you also add these to the `CUDA_EXAMPLES` below?

Reply:
done